### ISSUE TITLE
to render

### ISSUE THREAD

**Issue by lex111** [web-standards-ru/dictionary#356]
В словаре есть существительное `render` - отрисовка https://github.com/web-standards-ru/dictionary/blob/master/dictionary.md#render, соответственно, глагол, будет `отрисовать`? Гораздо чаще, как я замечал, используется `отрендерить`. Ещё неоднократно встречал употребление `отобразить`, хотя в таком случае пропадает различие между именно отрисовкой и отображением (например по какому-либо условие, наверное поэтому `render` как глагол нельзя переводить как `отобразить`?)

**Comment by SelenIT** [comment]
Согласен, _отобразить_ — это скорее _to display_. А _отрисовать_ и _отрендерить_, по-моему, можно использовать практически на равных (первое чуть-чуть формальнее).

**Comment by lex111** [comment]
> отрендерить

Калька, разве нет? Отрисовать — лучший вариант.
